imagersite/imager_images/forms.py

```diff
@@ -27,13 +29,13 @@ class Meta:
         """Meta data for Photo form."""
 
         model = Photo
-        fields = ['albums', 'title', 'description', 'image', 'published']
+        fields = ['title', 'description', 'image', 'published']
 
     def __init__(self, *args, **kwargs):
         """Init for Photo form."""
-        username = kwargs.pop('username')
+        # username = kwargs.pop('username')
         super().__init__(*args, **kwargs)
-        self.fields['albums'].queryset = Album.objects.filter(user__username=username)
+        # self.fields['albums'].queryset = Album.objects.filter(user__username=username)
```

Comment:
If there is nothing you are overwriting in your custom form class, it is not necessary to create a custom form class like this. Just specify the `fields` on your view instead of a `form_class`.